### ISSUE TITLE
TT tomography Multi CPU usage in Unix, how?

### ISSUE THREAD

**Issue by GGDRriedel** [gimli-org/gimli#403]
### Problem description

I want to find possiblilites to use multiple CPUs on Ubunti 20.04

### Your environment

**Operating system**:  Ubuntu 20.04

**pyGIMLi version**: 1.25
**Way of installation**: conda

We have a pretty beefy system with 96 Logical, 48 physical CPUs and 256 GB RAM

Some tomographies we do are in the multi-kilometer range.

On Windows I see pretty much all physical cores being used , on LINUX only one. 

Is there a way to enable multi CPU support on Linux?

![grafik](https://user-images.githubusercontent.com/48824691/176437900-38628557-112e-402b-a1bb-b399f72a167b.png)

**Comment by carsten-forty2** [comment]
Are there total runtime differences between Windows and Linux for your problem? Don't understand why are so much core used in Windows. The multi core calculation needs some rework but is currently low on priority.

**Comment by GGDRriedel** [comment]
Not really. But I am running a pretty large dataset, potentially even bigger and it would be a great imporovement to run calculations faster and in parallel.
I can't say why the task mngr in windows shows all cores used, but it does. 

I saw remnants of it in the source code so I thought I'd ask.

**Comment by halbmy** [comment]
closing due to inactivity. Nevertheless we can keep this issue as a discussion for the future. Right now this is on low priority.
